packages/client/src/env.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * React script probably couldn't detect the .env file when I tried it
+ * so I settled for this
+ */
+const REACT_APP_GOOGLE_API_KEY = `YOUR_API_KEY`;
```

Comment:
Can we use https://create-react-app.dev/docs/adding-custom-environment-variables/? Otherwise this is fine

Reply:
yeah I tried doing that and other variations of doing the same thing, but the variable kept showing up as undefined